crates/matrix-sdk-ui/src/timeline/mod.rs

```diff
@@ -104,8 +104,6 @@ pub struct RepliedToInfo {
     event_id: OwnedEventId,
     /// The sender of the event to reply to.
     sender: OwnedUserId,
-    /// The timestamp of the event to reply to.
-    timestamp: MilliSecondsSinceUnixEpoch,
     /// The content of the event to reply to.
     content: ReplyContent,
```

Comment:
Instead of the full `content`, I think this could store directly the thread information as an `Option`; then we could likely get rid of `ReplyContent` too, which would be nice!